src/main/kotlin/org/rust/ide/template/postfix/stringBasedPostfixTemplates.kt

```diff
@@ -73,7 +76,12 @@ class IterPostfixTemplate(provider: RsPostfixTemplateProvider) :
     StringBasedPostfixTemplate("iter", "for x in expr",
         RsTopMostInScopeSelector { it.implementsIntoIter || it.implementsIter }, provider) {
     override fun getTemplateString(element: PsiElement): String =
-        "for x in ${element.text} {\n\$END$\n}"
+        "for \$name$ in ${element.text} {\n     \$END$\n}"
+
+    override fun setVariables(template: Template, element: PsiElement) {
+        val name = MacroCallNode(SuggestVariableNameMacro())
```

Comment:
Unfortunately, we can't just copy kotlin implementation because `SuggestVariableNameMacro` is part of java support. So if IDE doesn't have java module (for example, CLion or PyCharm) it will fail with `NoClassDefFoundError` exception